crates/iceberg/src/catalog/mod.rs

```diff
@@ -438,6 +438,31 @@ impl TableUpdate {
     }
 }
 
+/// TableCreation represents the creation of a table in the catalog.
```

Comment:
TableCreation->ViewCreation

Reply:
Fixed in the new commit together with clips